k8s/k8s_util.go

```diff
@@ -1,305 +0,0 @@
-package k8s
-
-import (
-	"fmt"
-	"time"
-
-	"github.com/satori/go.uuid"
-
-	"k8s.io/kubernetes/pkg/api"
-	"k8s.io/kubernetes/pkg/api/unversioned"
-	"k8s.io/kubernetes/pkg/apis/extensions"
-	client "k8s.io/kubernetes/pkg/client/unversioned"
-	"k8s.io/kubernetes/pkg/labels"
-	"k8s.io/kubernetes/pkg/util/intstr"
-	"k8s.io/kubernetes/pkg/util/wait"
-)
-
-const (
-	slugBuilderName  = "deis-slugbuilder"
-	slugBuilderImage = "luizalabs/slugbuilder:git-923c9f8"
-	slugRunnerImage  = "luizalabs/slugrunner:git-044f85c"
```

Comment:
Are you sure about these?
In #154 I'm using these consts to avoid "magic strings"